SpatialGDK/Extras/schema/skeleton_entity.schema

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component SkeletonEntityManifest {
+	id = 9989;
+	list<EntityId> skeleton_entities_to_populate = 1;
+	list<EntityId> populated_skeleton_entities = 2;
+}
+
+component_set SkeletonEntityAuthComponentSet {
```

Comment:
This raises an interesting question about if we want to just have one component set per worker. To which the answer is probably whatever makes doing this kind of stuff easier.

But in principle if there's only one worker type that could ever be authoritative over something, we can bung it in the same component set that everything else on that worker is in.

Reply:
What eventually happened with this? It doesn't look like it's used.